docs/user-guide/src/quick-start.md

```diff
@@ -0,0 +1,642 @@
+environment. We will go through how to configure and deploy Ironic and Baremetal
+Operator.
+
+### Management cluster
+
+If you already have a Kubernetes cluster that you want to use, go ahead and use
+that. Please ensure that it is connected to the relevant networks so that Ironic
+can reach the BMCs and so that the BareMetalHosts can reach Ironic.
+
+If you do not have an cluster already, you can create one using kind. Please
+note that this is absolutely not intended for production environments.
+
+We will use the following configuration file for kind, save it as `kind.yaml`:
+
+```yaml
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    # Open ports for Ironic
+    extraPortMappings:
+      # Ironic httpd
+      - containerPort: 6180
+        hostPort: 6180
+        listenAddress: "0.0.0.0"
+        protocol: TCP
+      # Ironic API
+      - containerPort: 6385
+        hostPort: 6385
+        listenAddress: "0.0.0.0"
+        protocol: TCP
+      # Inspector API
+      - containerPort: 5050
+        hostPort: 5050
+        listenAddress: "0.0.0.0"
+        protocol: TCP
+```
+
+As you can see, it has a few ports forwarded from the host. This is to make
+Ironic reachable when it is running inside the kind cluster.
+
+Now go ahead and create the cluster:
+
+```bash
+kind create cluster --config kind.yaml
+```
+
+We will need to install cert-manager also. It will be used to manage the
+certificates for Ironic later.
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.3/cert-manager.yaml
+```
+
+### DHCP server
+
+The BareMetalHosts must be able to call back to Ironic when going through the
+inspection phase. This means that they must have IP addresses in a network where
+they can reach Ironic. We will set up a DHCP server for this purpose.
+
+Any DHCP server can be used for this. We will here use the ironic container
+image that incudes dnsmasq and some scripts for configuring it.
+
+Create a configuration file like this and save it as `dnsmasq.env`:
+
+```bash
+# Specify the MAC addresses (separated by ;) of the hosts we know about and want to use
+DHCP_HOSTS=80:c1:6e:7a:e8:10;80:c1:6e:7a:5a:a8
+# Ignore unknown hosts so we don't accidentally give out IP addresses to other hosts in the network
+DHCP_IGNORE=tag:!known
+# Listen on this IP (management computer)
+PROVISIONING_IP=192.168.0.150
+# Give out IP addresses in this range
+DHCP_RANGE=192.168.0.100,192.168.0.149
+```
+
+You can now run the DHCP server like this:
+
+```bash
+docker run --name dnsmasq --rm -d --net=host --privileged --user 997:994 \
+  --env-file dnsmasq.env --entrypoint /bin/rundnsmasq \
+  quay.io/metal3-io/ironic
+```
+
+### Image server
+
+In order to do anything useful, we will need a server for hosting disk images
+that can be used to provision the servers.
+
+Create a folder to hold the disk images:
+
+```bash
+mkdir disk-images
+```
+
+Download an image to use for testing:
```

Comment:
This is supposed to be a real baremetal user guide, right? Let's avoid cirros then. I'm impressed you actually found some hardware where it works, but on most it probably won't (missing drivers for NICs, RAID controllers, etc).

Reply:
Thanks for the heads up! I was not aware of this